content/rancher-prime/azure/contents.lr

```diff
@@ -8,7 +8,9 @@ body:
 
 1. A Rancher-compatible AKS cluster. Please see
    [Rancher support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-7-6/)
-   for more details.
+   for more details. Also, the Rancher Prime PAYG offering can only be installed onto clusters in regions where both the
```

Comment:
I would ditch `Also, `.... and just start with "The Rancher Prime"

Reply:
Done